graphql/queries/getLearningJournal.ts

```diff
@@ -1,24 +0,0 @@
-import { gql } from 'graphql-request'
```

Comment:
This is an unrelated change but necessary.
Query removed since it was duplicated and unused.